scripts/setup-centos.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+TARGET=$1
+
+# TODO: Assert TARGET is one of "el6" and "el7"
+
+echo "[Install] dependencies"
+sudo yum update
+
+sudo yum install -y linux-image-extra-$(uname -r)
+sudo yum install -y git curl wget
+
+echo "[Install] docker-compose"
+if [ "$TARGET" = "el6" ]; then
+  # el6 does not work with docker-compose 1.4+
```

Comment:
@warrenvw  Yeah, this and Kernel issues sounds like big waterblock with `EL6`.

If everything else works OK, I think we can just remove `centos/6` from the Vagrantfile list or throw an error message that EL6 is not supported to run Docker.

Considering that it's End of Production 2 phase for RHEL6: https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux#Product_life_cycle I think it's mid-safe for us to do something like that.

Eg. you can skip EL6 to avoid spending too much time on it.